File: exercicios/cash.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    //show me the money
    int cents;
    do
    {
        cents = get_int("How many cents of change? ");
    }
    while (cents < 0 || cents > 100);

    //Calculete the change owed
    int quarters = cents / 25;
    int rest_quarters = cents % 25;
    int dimes = rest_quarters / 10;
    int rest_dimes = rest_quarters % 10;
    int nickels = rest_dimes / 5;
    int pennies = rest_dimes % 5;
    int coins = quarters + dimes + nickels + pennies;
    printf("Total of %i, %i coins of quarters, %i coins of dimes, %i coins of nickles, and %i coins of pennies\n", coins, quarters, dimes, nickels, pennies);
}

File: problems_set/pset6/cash.py
while True:
    cents = float(input("Change owed: "))
    if cents > 0:
        break

cents = round(cents * 100)
quarters = cents // 25
rest_quarters = cents % 25
dimes = rest_quarters // 10
rest_dimes = rest_quarters % 10
nickels = rest_dimes // 5
pennies = rest_dimes % 5

coins = quarters + dimes + nickels + pennies

print(f'{coins}')